packages/wdio-static-server-service/src/launcher.ts

```diff
@@ -10,14 +10,27 @@ const log = logger('@wdio/static-server-service')
 
 const DEFAULT_LOG_NAME = 'wdio-static-server-service.log'
 
+export interface MiddleWareOption{
```

Comment:
can you please re-format the file properly?

Reply:
Done...Please check